apps/o2-blocks/package.json

```diff
@@ -19,7 +19,7 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prepublish": "npm run clean",
-		"prepare": "webpack --config='../calypso-build/webpack.config.js' --output-path='./dist' editor='./src/editor.js' "
+		"prepare": "webpack --config='../../packages/calypso-build/webpack.config.js' --output-path='./dist' editor='./src/editor.js' "
```

Comment:
if we make this something other than `prepare`, it would naturally be ignored by the post-install prepare at the root

Reply:
I think renaming it to `build` would work

Reply:
Packages `prepare` apps `build`… works for me 👍